Privacy.md

```diff
@@ -0,0 +1,31 @@
+#Privacy 
+This is a plain English summary of all of the components within Piwik which may affect your privacy in some way. Please keep in mind that if you use third party Themes or Apps with Piwik, there may be additional things not listed here.
+Each of the items listed in this document can be disabled via Piwik's config.js file. 
+You can read about Piwik privacy policy [here](http://piwik.org/privacy-policy/).
+##Privacy for Piwik admins and website owners
+As admins is the Piwik admin and any other user acessing Piwik
```

Comment:
In this section, we document how you can configure your Piwik service to better protect your privacy.